source/extensions/extensions_metadata.yaml

```diff
@@ -1524,9 +1524,10 @@ envoy.matching.inputs.application_protocol:
   - envoy.extensions.matching.common_inputs.network.v3.ApplicationProtocolInput
 envoy.matching.inputs.filter_state:
   categories:
+  - envoy.matching.http.input
   - envoy.matching.network.input
   security_posture: unknown
-  status: stable
+  status: alpha
```

Comment:
Is this change allowed? update the status from `stable` to `alpha`? cc @phlax

Reply:
This is similar to the problem we had with upstream/downstream filters requiring different classification. Maybe we need to change this manifest to have a status per category membership?

Reply:
Yeah, no downgrades like this. Maybe fine grained classification could make sense, but like any feature development, once a filter is stable, it should probably remain so. We already have finer grained classification for security posture (robust to downstream vs. upstream).

Reply:
In this case may I know how should I update extensions_metadata?

Reply:
I would leave it as stable for now and ensure we continue to treat the new category as stable if that works.